Grammar/python.gram

```diff
@@ -1297,6 +1299,17 @@ invalid_raise_stmt:
 invalid_del_stmt:
     | 'del' a=star_expressions {
         RAISE_SYNTAX_ERROR_INVALID_TARGET(DEL_TARGETS, a) }
+invalid_assert_stmt:
+    | 'assert' a=expression '=' b=expression {
+        RAISE_SYNTAX_ERROR_KNOWN_RANGE(
+            a, b, 
+            "cannot assign to %s here. Maybe you meant '==' instead of '='?",
+            _PyPegen_get_expr_name(a)) }
+    | 'assert' expression ',' a=expression '=' b=expression {
+        RAISE_SYNTAX_ERROR_KNOWN_RANGE(
+            a, b, 
+            "cannot assign to %s here. Maybe you meant '==' instead of '='?",
+            _PyPegen_get_expr_name(a)) }
```

Comment:
Is the second alternative here needed in reality. If someone's using the second argument here and have a comparison there, maybe they're confused and meant something completely different?